test/units/util.sh

```diff
@@ -216,3 +216,19 @@ kernel_supports_lsm() {
 
     return 1
 }
+
+MOUNTED_USR_OVERLAY=false
+
+maybe_mount_usr_overlay() {
+    if [ ! -w /usr ]; then
+        mkdir -p /var/usr-overlay/{upperdir,workdir}
```

Comment:
Also, please use `[[ ... ]]`.

Reply:
It was useful for debugging locale tests, but since we're hoping to make the disk images ephemeral I can't rely on that forever, so I may as well change it now and get used to debugging without it.